app/backend/src/couchers/models.py

```diff
@@ -167,6 +179,13 @@ class User(Base):
     joined = Column(DateTime(timezone=True), nullable=False, server_default=func.now())
     last_active = Column(DateTime(timezone=True), nullable=False, server_default=func.now())
 
+    heard_about_us_source = Column(
+        Enum(HeardAboutUs),
+        server_default=text("'source_friend'"),  # Use the string representation of the Enum value
+        nullable=False,
+    )
```

Comment:
Can't it just be:

```suggestion
    heard_about_us_source = Column(Enum(HeardAboutUs), nullable=True)
```

That seems more in line with our existing pattern. Also for existing users we have no way of knowing how they heard about us. Which means it needs to be nullable as there's no way we can backfill it.

I also don't think we should require this to sign up. It's more a "nice to have".

Reply:
ok changes to nullable = True, i was getting a pre hook error for not having a server default value